modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/ProcessStore.java

```diff
@@ -1087,10 +1088,10 @@ public String deletePredecessor(String deletePredecessor){
                     reg.put(processAssetPath, resource);
                 }
             }
-        }catch (Exception e){
+        }catch (Exception e) {
             log.error(e);
```

Comment:
Please add a descriptive error message when logging the error. 
log.error("Error occurred when deleting predecessor", e); 
Also better to catch the specific error rather than catching the super Exception object.